site/prepare.md

```diff
@@ -32,6 +32,8 @@ You can then customize the `k8s_variable.properties` and `create_k8s_secrets.sh`
 
 The '-target' parameter is referring to a file on the file system '$WLSDEPLOY_HOME/lib/target/<target value>/target.json'
 
+A target may also be configured in a [Custom Configuration](config/custom_configuration.md) directory, in `$WDT_CUSTOM_CONFIG/target/<target name>/target.json`. 
+
```

Comment:
Two other items of note to be corrected in prepare.md:
json -> JSON (please correct this globally)
In the output directory, you will find -> In the output directory, you will find: (stem sentences required a trailing colon.)